internal/controller/sync/sync.go

```diff
@@ -99,82 +75,34 @@ func handlePVCState(ctx context.Context, nfspvc danaiov1alpha1.NfsPvc, k8sClient
 		}
 	}
 
-	if pvc.Status.Phase == corev1.ClaimLost { // if the pvc's phase is 'lost', so probably the associated pv was deleted. In order to fix that the "bind" annotation needs to be deleted.
-		bindStatus, ok := pvc.ObjectMeta.Annotations[PvcBindStatusAnnotation]
-		if ok && bindStatus == "yes" {
-			// Use retry on conflict to update the PVC.
-			err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-				if err := k8sClient.Get(ctx, types.NamespacedName{Namespace: nfspvc.Namespace, Name: nfspvc.Name}, &pvc); err != nil {
-					return err
-				}
-				delete(pvc.ObjectMeta.Annotations, PvcBindStatusAnnotation)
-				updateErr := k8sClient.Update(ctx, &pvc)
-				if errors.IsConflict(updateErr) {
-					// Conflict occurred, let's re-fetch the latest version of PVC and retry the update.
-					if getErr := k8sClient.Get(ctx, types.NamespacedName{Namespace: nfspvc.Namespace, Name: nfspvc.Name}, &pvc); getErr != nil {
-						return getErr
-					}
-				}
-				return updateErr
-			})
-			return err
-		}
-		return nil
+	if pvc.Status.Phase == corev1.ClaimLost { // if the pvc's phase is 'lost', so probably the associated pv was deleted
```

Comment:
Don't use comments like this, always in a new line

Reply:
omers fault